src/VoyagerServiceProvider.php

```diff
@@ -88,6 +91,11 @@ public function boot(Router $router, Dispatcher $event)
             $this->addStorageSymlinkAlert();
         });
 
+        //Added a listener on event after update data
+        $event->listen('voyager.insert_update_data', function ($request, $slug, $rows, $data) {
```

Comment:
So... we're calling `cropImages()` for ALL insert/update calls, regardless of whether images are used at all? I'm not sure I like that.  Was this tested in those cases?